src/cloud/components/Comments/ThreadItem.tsx

```diff
@@ -94,6 +105,79 @@ function ThreadItem({
     return threadComments[0].user
   }, [threadComments, users])
 
+  const threadCommentReplyInfo = useCallback(() => {
+    if (
+      threadComments &&
+      (threadComments.length > 1 ||
+        (threadComments.length == 1 && thread.initialComment == null))
+    ) {
+      return (
+        <div className={'thread__comment__line_more_replies_container'}>
+          <div
+            onClick={() => onSelect(thread)}
+            className={'thread__comment__line__replies__link'}
+          >
+            {translate(lngKeys.ThreadReplies, {
+              count: thread.commentCount,
```

Comment:
I think replies should be for an initial comment. So I think we should not count the initial comment.

Reply:
Yes, fixed in another PR, rebase will fix this